Templates/colors.json

```diff
@@ -0,0 +1,4687 @@
+        { "name": "Unknown220", "type": "uint32" },
+        { "name": "Unknown221", "type": "float32" },
+        { "name": "Unknown222", "type": "float32" },
+        { "name": "Unknown223", "type": "float32" },
+        { "name": "Unknown224", "type": "float32" },
+        { "name": "Unknown225", "type": "float32" },
+        { "name": "Unknown226", "type": "uint32" },
+        { "name": "Unknown227", "type": "float32" },
+        { "name": "Unknown228", "type": "uint32" },
+        { "name": "Unknown229", "type": "uint32" },
+        { "name": "Unknown230", "type": "float32" },
+        { "name": "Unknown231", "type": "float32" },
+        { "name": "Unknown232", "type": "float32" },
+        { "name": "Unknown233", "type": "uint32" },
+        { "name": "Unknown234", "type": "float32" },
+        { "name": "Unknown235", "type": "uint32" },
+        { "name": "Unknown236", "type": "uint32" },
+        { "name": "Unknown237", "type": "float32" },
+        { "name": "Unknown238", "type": "float32" },
+        { "name": "Unknown239", "type": "uint32" },
+        { "name": "Unknown240", "type": "float32" },
+        { "name": "Unknown241", "type": "float32" },
+        { "name": "Unknown242", "type": "float32" },
+        { "name": "Unknown243", "type": "float32" },
+        { "name": "Unknown244", "type": "float32" },
+        { "name": "Unknown245", "type": "float32" },
+        { "name": "Unknown246", "type": "float32" },
+        { "name": "Unknown247", "type": "float32" },
+        { "name": "Unknown248", "type": "uint32" },
+        { "name": "Unknown249", "type": "uint32" },
+        { "name": "Unknown250", "type": "float32" },
+        { "name": "Unknown251", "type": "float32" },
+        { "name": "Unknown252", "type": "float32" },
+        { "name": "Unknown253", "type": "float32" },
+        { "name": "Unknown254", "type": "uint32" },
+        { "name": "Unknown255", "type": "uint32" }
+      ]
+    },
+    "BossEggMobileSpawner": {
+      "fields": [
+        { "name": "Datum", "type": "BossEggMobileSpawner::Datum" },
+        { "name": "Altitude", "type": "float32" },
+        { "name": "Unknown1", "type": "uint8" },
+        { "name": "Unknown2", "type": "uint8" },
+        { "name": "AppearOffset1D", "type": "float32" },
+        { "name": "Speed", "type": "float32" },
+        { "name": "Unknown3", "type": "uint8" }
+      ]
+    },
+    "BossFerrisWheelBaseSpawner": {
+      "fields": [
+        { "name": "SearchRadius", "type": "float32" }
+      ]
+    },
+    "BossFerrisWheelSpawner": {
+      "parent": "BossFerrisWheelBaseSpawner",
+      "fields": [
+        { "name": "ShotSpeed", "type": "float32" },
+        { "name": "ShotLifeTime", "type": "float32" },
+        { "name": "ShotHomingWait", "type": "float32" },
+        { "name": "ShotChaseTime", "type": "float32" },
+        { "name": "ShotRotateSpeed", "type": "float32" },
+        { "name": "ShotCollisionRadius", "type": "float32" }
+      ]
+    },
```

Comment:
Parameters named based on in-game functionality and other objects with similar functionality and parameter names.
`ShotRotateSpeed` is given a completely custom name as no other parameter matches its functionality, however it mimics Sonic Team's naming scheme as best as it could. `ShotHomingWait` is taken from Sonic Forces.